pkg/move/technologies.go

```diff
@@ -25,9 +25,15 @@ type FileEntry struct {
 	MD5     string `json:"md5"`
 }
 
-var _ copyFunc = copyByTechnology
+var _ copyFunc = CopyByTechnologyWrapper("test")
```

Comment:
Why `"test"`? Wouldn't an empty string suffice? I guess this is some kind of default?